cosmos-gui/src/app.js

```diff
@@ -35,6 +35,7 @@ var mysqlDriver = require('./mysql_driver.js');
 var OAuth2 = require('./oauth2').OAuth2;
 var logger = require('./logger.js');
 var appUtils = require('./app_utils.js');
+var constants = require('constants');
```

Comment:
Hi @frbattid , I am sorry but where is this new `contants.js` or `constants.json` file located? I cannot find it... :)